### PR TITLE
Feature e6 modules

### PR DESCRIPTION
**Pull request by mutesihope**
**In this PR I have**
-	Used ES6 syntax.
•	Divided the code into modules.
o	Created a separate file for each module in the modules directory.
o	Imported modules into the index.js file.
•	Kept all functionalities of the app without errors.
•	Refactored functions to arrow functions.

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @mutesihope,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project with failed linter checks.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import './style.css';
+import BookManager from '../modules/BookManager.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+  BookManager();
+});
```
Comment (Olamarx) [inline src/index.js:6]: - [ ] I can see you are calling a class here, note that classes aren't called without instantiating it first, so you will have to declare it like `const bookmanager = new BookManager()` then you can now call the declared bookmanager.
- [ ] I also noticed you are calling the whole class, you have methods in the class, so untill you call those methods by dot notation because classes are OOP. Let me know if something is unclear.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
```
Comment (Olamarx) [inline src/index.html:6]: - [OPTIONAL] When you set up a project with Webpack, it's a common practice to bundle your CSS and JavaScript files using Webpack's bundling capabilities. This means that instead of manually adding individual CSS and JavaScript files to your HTML, you would include a single bundled file that contains all the necessary styles and scripts.

Here's how the process typically works:

Webpack Configuration: In your Webpack configuration file (e.g., webpack.config.js), you'll define entry points for your application, specify how your files should be processed (using loaders for CSS, JavaScript, etc.), and set output paths for the generated bundles.

Bundling Process: When you run the Webpack build command, it will process your entry points, gather dependencies, apply loaders (for transpilation, minification, etc.), and create bundled output files.

Including Bundles in HTML: After the bundling process is complete, you'll include the generated bundles in your HTML file. This usually involves adding a <script> tag for the JavaScript bundle and a <link> tag for the CSS bundle.

So, in short, you don't manually add individual CSS and JavaScript files to your HTML. Instead, you include the bundled files that Webpack generates. This has several advantages, including optimized loading and better management of dependencies.

File: modules/BookManager.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    this.addBookLink.addEventListener('click', this.showSection.bind(this, 'addBookSection'));
+    this.contactLink.addEventListener('click', this.showSection.bind(this, 'contactSection'));
+
+    this.bookForm.addEventListener('submit', this.addBook.bind(this));
+    this.displayBooks();
+  }
+
+  static loadBooks() {
+    const storedBooks = localStorage.getItem('books');
+    return storedBooks ? JSON.parse(storedBooks) : [];
```
Comment (Olamarx) [inline modules/BookManager.js:22]: - [ ] I don't know why you used static here but it doesn't load the pages until I removed the static used the this keyword on the storedBooks. Kindly fix that.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @mutesihope, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- ES6 usage ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] I also could not find anything as to Luxon for the date and time, you must have forgotten, Kindly work on it.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
